RELEASE.md

```diff
@@ -0,0 +1,94 @@
+# Rusk Monorepo Release Process
+
+## **1. Branching Strategy**
+
+### `master` Branch
+- **Purpose:** Active development.
+- **Policy:**
+  - All new features behind feature flags.
+  - Merged only after passing CI, integration tests, and approval.
+
+### `devnet` Branch
+- **Purpose:** Weekly snapshots from `master` for early testing.
+- **Policy:** Updated weekly, deploying directly to the Devnet environment.
+
+### Release Branches (`rusk-release-vX.Y`)
+- **Purpose:** Stable releases for mainnet or testnet deployments.
```

Comment:
I am a bit confused, especially with the **7. Environment Versioning Policy** stating that testnet is beta (well, I think it should be RC, see the comments there).

If so, testnet should be for the rc branches, while the mainnet for the stable branches (rusk-release-vX.Y) only, no?